tools/benchmark_tool/openvino/tools/benchmark/utils/inputs_filling.py

```diff
@@ -127,6 +132,9 @@ def get_input_data(paths_to_input, app_input_info):
 
 
 def get_image_tensors(image_paths, info, batch_sizes):
+    if cv2 is None:
+        raise ImportError('opencv required for image processing in benchmark_app, please install it')
```

Comment:
Can we provide more descriptive message what to do? E.g. some not experienced users cannot understand that it's even a python package.
Maybe something like ", please, install opencv-python or opencv-python-headless using pip"?